### ISSUE TITLE
Publish .box to vagrantcloud.com

### ISSUE THREAD

**Issue by carmstrong** [coreos/coreos-vagrant#83]
Right now, once the alpha box is downloaded, Vagrant will never see it as outdated - it will continue to happily run it. If a box is downloaded from https://vagrantcloud.com, however, vagrant will be able to detect when a new version is published and can prompt the user. 

Can the coreos team begin publishing images to https://vagrantcloud.com?

**Comment by carmstrong** [comment]
@marineam Could really use the ability for http://stable.release.core-os.net/amd64-usr/current/coreos_production_vagrant.json to publish *all* versions, not the latest one. We are having to revert to an older CoreOS for Docker 1.5 due to https://github.com/docker/docker/issues/9997, and specifying a specific version in the Vagrantfile results in:

```console
$ vagrant up
Bringing machine 'deis-01' up with 'virtualbox' provider...
Bringing machine 'deis-02' up with 'virtualbox' provider...
Bringing machine 'deis-03' up with 'virtualbox' provider...
==> deis-01: Running triggers before up...
==> deis-01: Box 'coreos-stable' could not be found. Attempting to find and install...
    deis-01: Box Provider: virtualbox
    deis-01: Box Version: = 647.2.0
==> deis-01: Loading metadata for box 'http://stable.release.core-os.net/amd64-usr/current/coreos_production_vagrant.json'
    deis-01: URL: http://stable.release.core-os.net/amd64-usr/current/coreos_production_vagrant.json
The box you're attempting to add has no available version that
matches the constraints you requested. Please double-check your
settings. Also verify that if you specified version constraints,
that the provider you wish to use is available for these constriants.

Box: coreos-stable
Address: http://stable.release.core-os.net/amd64-usr/current/coreos_production_vagrant.json
Constraints: = 647.2.0
Available versions: 717.1.0
```

The JSON payload only lists the latest version:
```json
{
  "name": "coreos-stable",
  "description": "CoreOS stable",
  "versions": [{
    "version": "717.1.0",
    "providers": [{
      "name": "virtualbox",
      "url": "http://stable.release.core-os.net/amd64-usr/717.1.0/coreos_production_vagrant.box",
      "checksum_type": "sha256",
      "checksum": "42f994a44870fd65fb993a09532f2f11ebb078506d07ee4864ec67635c573a4a"
    }]
  }]
}
```

**Comment by crawford** [comment]
Moved to https://github.com/coreos/bugs/issues/1382
